### PR TITLE
User - People UI - Redesign

### LINKED ISSUE #1607: USER REDESIGN: redesign the People screen for the talawa user portal
**Is your feature request related to a problem? Please describe.**
we are planning redesigning the user portal, this issue is a part of that redesigning.

**Describe the solution you'd like**
we want to redesign the People screen for the user portal and follow a theme similar to the talawa admin portal.

**Screenshots**
- Attaching the screenshots for reference.

current situation:
Screenshot 2024-02-21 at 1 04 25 AM

Desired Result:
Screenshot 2024-02-21 at 12 58 59 AM

Please refer the figma file for a detailed understanding : 
https://www.figma.com/file/WksD4uIa5sQJMIk3pR4l0c/Talawa-Design-System-(Official)?type=design&node-id=3447%3A59&mode=design&t=fkgMPaaz67qLwmMz-1

**_Related issue_** : #1510 (sidenavbar is covered in this issue)

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by pranshugupta54**
**What kind of change does this PR introduce?**

User -> People -> Redesigning

**Issue Number:**

Fixes #1607 

**Did you add tests for your changes?**

**Snapshots/Videos:**

image

**Tests:**
image
image

**Other information**

- The left side panel (according to Figma) should be fixed in another issue to match the design.
- Added extra Avatar (as per current codebase, `no avatar was there in new design`)

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by pranshugupta54** [comment]
Not sure why `Check for eslint-disable` is failing, because this PR doesn't even touch OrganizationPeople folder
image

**Comment by pranshugupta54** [comment]
Not sure why `Runs Introspection on the GitHub talawa-api repo on the schema.graphql file` fails. I've only added `userType` for `ORGANIZATIONS_MEMBER_CONNECTION_LIST` to display roles for the user on People page. Github Action shows some irrelevant error.

image

**Comment by pranshugupta54** [comment]
Added `userType` in tests

**Comment by palisadoes** [comment]
Please remove the lint ignore statement so that we can review
```
talawa-admin/src/screens/OrganizationPeople/OrganizationPeople.tsx
```

**Comment by palisadoes** [comment]
Fix the failing tests so that we can review

**Comment by pranshugupta54** [comment]
Hey @palisadoes, the eslint-disable directive is already present in the original repository. This file is not associated with the current pull request. It was included in a previous pull request, PR #1568.

**Comment by pranshugupta54** [comment]
I believe the GitHub Action failure for all upcoming PRs is being caused by a previously merged PR that included the eslint-disable directive.

**Comment by palisadoes** [comment]
1. Please fix it. 
2. This is a newly implemented test, there should be a few that we missed

**Comment by Cioppolo14** [comment]
@Tajcore @noman2002 Can you review this PR?

File: public/locales/en.json [hunk 1]
```diff
@@ -634,10 +634,11 @@
     "allOrganizations": "All Organizations",
     "joinedOrganizations": "Joined Organizations",
     "createdOrganizations": "Created Organizations",
-    "search": "Search",
+    "search": "Search users",
     "nothingToShow": "Nothing to show here.",
```
Comment (noman2002) [inline public/locales/en.json:638]: Please update this in all languages.

**Review by noman2002 — CHANGES_REQUESTED** [review]

File: public/locales/en.json (hunk 1 above)
Comment (pranshugupta54) [reply public/locales/en.json:638]: fixed

**Comment by Tajcore** [comment]
Please update branch and resolve any merge conflicts that may arise

**Comment by aashimawadhwa** [comment]
Can you relove pagination? @pranshugupta54  we are suing infinite scroll across talawa .

**Comment by pranshugupta54** [comment]
> Can you remove pagination? @pranshugupta54 we are using infinite scroll across talawa .

Hey @aashimawadhwa, can you please clarify over this. I see pagination everywhere across Talawa. Can you refer some infinite scroll? Do you want me to list all the users at once or do you want them to load depending on user scroll event? (In my opinion, pagination would be a better choice here. Infinite scroll is preferred when there is not definite end point or count, like scrolling feed section with endless posts. But user list is a finite number so pagination should be preferred.)

**Comment by pranshugupta54** [comment]
A recent merge made like 100+ changes to file, not sure whats the issue. Just made a new PR for this #1694

**Pull request closed without merging**